packages/@okta/vuepress-site/docs/release-notes/2019-09-0/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2019.09.0
+
+| Change                                                                                                        | Expected in Preview Orgs |
+|---------------------------------------------------------------------------------------------------------------|--------------------------|
+| [Features API is Early Access EA in Preview and Production](#features-api-is-ea)                              | September 4, 2019        |
+| [Mappings API is now Generally Available (GA) in Production](#mappings-api-is-now-ga-in-production)           | September 4, 2019        |
+| [Error Object in SAML Assertion Inline Hook](#error-object-in-saml-assertion-inline-hook)                     | September 4, 2019        |
+| [Rate Limits for /oauth2 Endpoints](#rate-limits-for-oauth2-endpoints)                                        | September 4, 2019        |
+| [Rate Limits for Authorization Server Public Metadata](#rate-limits-for-authorization-server-public-metadata) | September 4, 2019        |
+| [Bug Fixed in 2019.09.0](#bug-fixed-in-2019-08-0)                                                             | September 4, 2019        |
+
+### Features API is Early Access EA in Preview and Production
+
+The [Features API](/docs/reference/api/features/) provides operations to manage self-service Early Access (EA) features in your Production and Preview orgs, and Beta features in your Preview org. <!-- OKTA-241445 -->
```

Comment:
Don't need 'Early Access (EA)'....since you introduced EA in the title - you can just use EA here.

Also, the comma isn't needed before and